src/compose/service-manager.ts

```diff
@@ -611,7 +611,7 @@ async function prepareForHandover(service: Service) {
 	const container = docker.getContainer(svc.containerId);
 	await container.update({ RestartPolicy: {} });
 	return await container.rename({
-		name: `old_${service.serviceName}_${service.imageId}_${service.imageId}_${service.releaseId}`,
+		name: `old_${service.serviceName}_${service.imageId}_${service.commit}`,
```

Comment:
I see that service.releaseId is omitted in the container name here, but not on line 154. Is this intended?

Reply:
Nice catch, that's an error